_data/program.yaml

```diff
@@ -173,4 +173,28 @@ levels:
         url: https://www.choseizen.org/virtual-dojo
       - label: Conversational Randori to prepare for Edge challenge
         url: https://aikidodiscovery.net/randori-the-art-of-taking-control-in-chaos/
+relationships:
+  - label: Kōhai
+    intention: To learn from and defer to Senpai and Sensei
+    aspects:
+      - label: Attend the Mat regularly
+      - label: Practice the forms daily
+      - label: Demonstrate what you have learned through creating Mindset Dojo Insights articles
+      - label: Treat others with respect and kindness
+  - label: Senpai
+    intention: To mentor Kōhai and learn from Sensei
+    aspects:
+      - label: Lead the Mat regularly
+      - label: Practice the forms daily
+      - label: Mentor Kōhai through their learning journey
+      - label: Demonstrate what you have learned through creating Mindset Dojo Insights articles
+      - label: Treat others with respect and kindness
+  - label: Sensei
+    intention: To lead the dojo and mentor Senpai and Kōhai
```

Comment:
There is an attitude to the ecosystem that is not expressing here ... always learning through every moment